src/Documents.jl

```diff
@@ -253,14 +253,9 @@ function Document(;
         sitename :: AbstractString   = "",
         authors  :: AbstractString   = "",
         analytics :: AbstractString  = "",
-        version :: AbstractString    = "",
-        html_prettyurls  :: Bool     = true,
-        html_disable_git :: Bool     = false,
-        html_edit_branch :: Union{String, Nothing} = "master",
-        html_canonical   :: Union{String, Nothing} = nothing,
-        others...
+        version :: AbstractString    = ""
     )
-    Utilities.check_kwargs(others)
+    # Utilities.check_kwargs(others)
```

Comment:
Let's keep this in for now. Theoretically, we currently allow passing arbitrary arguments without throwing an error, so this would be breaking.